site/en/docs/extensions/mv3/xhr/index.md

```diff
@@ -1,18 +1,13 @@
 ---
 layout: "layouts/doc-post.njk"
-title: "Cross-origin XMLHttpRequest"
-seoTitle: "Chrome Extensions: Cross-origin XMLHttpRequest"
+title: "Cross-origin Network Requests"
+seoTitle: "Chrome Extensions: Cross-origin Network Requests"
 date: 2012-09-18
-updated: 2020-03-09
-description: How to implement cross-origin XHR in your Chrome Extension.
+updated: 2023-05-23
+description: How to implement cross-origin network requests in your Chrome Extension.
 ---
 
-{% Aside 'warning' %}
-In Manifest V3, `XMLHttpRequest` is not supported in background pages (provided by Service Workers).
-Please consider using its modern replacement, `fetch()`.
-{% endAside %}
-
-Regular web pages can use the [`XMLHttpRequest`][1] object to send and receive data from remote
+Regular web pages can use the [`XMLHttpRequest`][1] or [fetch()][13] objects to send and receive data from remote
```

Comment:
Can we switch the order of fetch and XMLHttpRequest to indicate some preference? And maybe say APIs rather than "objects".